ECOSYSTEM.md

```diff
@@ -87,6 +87,7 @@ If your project isn't listed here and you would like it to be, please feel free
 - [randoku](https://github.com/stchris/randoku): A tiny web service which generates random numbers and shuffles lists randomly
 - [sero](https://github.com/clowzed/sero): Host static sites with custom subdomains as surge.sh does. But with full control and cool new features. (axum, sea-orm, postgresql)
 - [Hatsu](https://github.com/importantimport/hatsu): 🩵 Self-hosted & Fully-automated ActivityPub Bridge for Static Sites.
+- [Mini RPS](https://github.com/marcodpt/minirps): Mini reverse proxy server, with no deps, HTTPS, CORS, static file hosting and template engine (minijinja).
```

Comment:
What do you mean by no deps? Clearly you depend on axum, minijinja 😅

Reply:
Sorry, I'll fix it 

I wanted a single binary without OS library dependencies

Reply:
I'd describe that as "no C dependencies". I think that's the case for most projects on the list so I'll merge as-is without a special note on this. Thanks!